pkg/ebpf/tracee.go

```diff
@@ -190,6 +195,10 @@ func (t *Tracee) handleEventsDependencies(eventId events.ID, submitMap uint64) {
 		}
 		ec.submit |= submitMap
 		t.events[dependentEvent.EventID] = ec
+
+		if isSignatureEvent {
+			t.eventSignatures[dependentEvent.EventID] = true
```

Comment:
Can't we just initialize this map above and get rid of the isSignatureEvent variable?

Reply:
didn't understand, can you expand?

Reply:
seems like this was fixed in the recent push